src/sysusers/sysusers.c

```diff
@@ -64,7 +64,9 @@ typedef struct Item {
         uid_t uid;
 
         bool gid_set:1;
-        bool gid_must_exist:1;
+        // id_set_strict means that the gid must exist and that
```

Comment:
'the gid must exist' sounds slightly strange. How about 'the group with the specified gid must exist'?